packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -141,30 +153,50 @@
 />
 
 <aside class="on-this-page" bind:this={containerEl}>
-	<h2>On this page</h2>
-	<nav aria-label="On this page">
-		<ul>
-			<li>
-				<a
-					href="{base}/docs/{details.slug}"
-					class:active={hash === ''}
-					on:click={() => dispatch('select')}>{details.title}</a
-				>
-			</li>
-			{#each details.sections as { title, slug }}
+	<button class="heading" on:click={() => (mobile_menu_open = !mobile_menu_open)}>
+		<span class="h2">On this page</span>
+
+		<span class="expand-icon" class:inverted={mobile_menu_open}>
+			<Icon name="chevron-down" />
+		</span>
+	</button>
+
+	{#if (browser && !$is_mobile) || ($is_mobile && mobile_menu_open)}
+		<nav
+			aria-label="On this page"
+			transition:slide={{ axis: 'y', easing: expoOut, duration: $reduced_motion ? 0 : 400 }}
+		>
+			<ul>
 				<li>
 					<a
-						href={`#${slug}`}
-						class:active={`#${slug}` === hash}
-						on:click={() => dispatch('select')}>{title}</a
+						href="{base}/docs/{details.slug}"
+						class:active={hash === ''}
+						on:click={() => dispatch('select')}>{details.title}</a
 					>
 				</li>
-			{/each}
-		</ul>
-	</nav>
+				{#each details.sections as { title, slug }}
+					<li>
+						<a
+							href={`#${slug}`}
+							class:active={`#${slug}` === hash}
+							on:click={() => dispatch('select')}
+						>
+							{title}
+						</a>
+
+						<hr />
+					</li>
+				{/each}
+			</ul>
+		</nav>
+	{/if}
 </aside>
 
 <style>
+	* {
+		box-sizing: border-box;
+	}
```

Comment:
do we really need this? I'd expect this to be set globally

Reply:
Oops yeah, left it while tryna debug something

Reply:
Removed